komodo/yaml_file_type.py

```diff
@@ -1,9 +1,49 @@
 import argparse
 import os
+from collections import namedtuple
 from pathlib import Path
-from typing import Any, Dict
+from typing import Dict, List
 
-import yaml as yml
+from ruamel.yaml import YAML
+
+komodo_error = namedtuple(
```

Comment:
I am a little bit unsure which casing to choose for this. Module constants should be upper-cased, but this is more like a class definition, should we case it as such?